components/organisms/o-my-account-shipping-details.vue

```diff
@@ -190,12 +202,21 @@ export default {
         country: '',
         telephone: ''
       },
-      countries: Countries
+      countries: Countries,
+      states: States
     }
   },
   computed: {
     addresses () {
       return this.$store.state.user.current.addresses
+    },
+    stateOptions () {
+      let countryCode = this.editedAddress.country ? this.countries.find(country => country.name === this.editedAddress.country).code : config.i18n.defaultCountry
+      if (this.states[countryCode] !== null) {
+        return this.states[countryCode]
```

Comment:
by changing that to `return this.states[countryCode] !== null ? this.states[countryCode] : []` we could save a line :) what if `this.states[countryCode]` is `undefined`? for that you could remove `!== null` and it would return empty array for each falsy value.

Reply:
Make sense, will make the necessary changes, with the change log ;)

Reply:
cool, this code appears 3 times in this PR so please adjust all of them :)

Reply:
Hi @RakowskiPrzemyslaw , I have made these changes, please review!
Added change log entry, under ## [1.0.3].
Please make sure this get merge after https://github.com/DivanteLtd/vue-storefront/pull/4680 as this provide data for the dropdown

Reply:
@AishwaryShrivastav okay, please update the branch for that PR and add the changelog and I will merge it :)

Reply:
and change the target branch to current hotfix

Reply:
Done